ch05construction/conventions.py

```diff
@@ -1,81 +0,0 @@
-### "dense"
```

Comment:
Is this file only being used in `02conventions`?

Reply:
Can confirm this is not being used anywhere. In fact, it is not being used in `02conventions` too. The variables in `02conventions` are coming from `context.py` -

https://github.com/UCL/rsd-engineeringcourse/blob/6741f65bbfdfbe09a755dafbbeff5ae879f17e7f/ch05construction/01introduction.ipynb.py#L133-L188

Reply:
Looking at this again, maybe I should not get rid of `from context import *`, given that it is present in a few other file as well and removing it from everywhere will result in a lot of repetition. Maybe I should just import specific things from the file and assume that the readers are going through the course serially so they know what `context.py` is.